homeassistant/components/devolo_home_network/config_flow.py

```diff
@@ -78,6 +78,12 @@ async def async_step_zeroconf(
         self, discovery_info: zeroconf.ZeroconfServiceInfo
     ) -> FlowResult:
         """Handle zeroconf discovery."""
+        if (
+            "MT" not in discovery_info.properties
```

Comment:
It's not totally clear to me but perhaps we can update the manifest to require a match on these keys?

https://developers.home-assistant.io/docs/creating_integration_manifest#zeroconf

Reply:
I tried it by adding 

```
"zeroconf": [{"type":"_dvl-deviceapi._tcp.local.", "properties":{"MT":"*"}}],
```

but I still run into the case, where the properties are empty.

Reply:
And you ran the hassfest script to update the generated zeroconf.py file before testing?

Reply:
don't tell anybody ....

I changed it to that approach. Thanks for the hint!